### ISSUE TITLE
 Item transactions within a grouped expanded row can be paginated?

### ISSUE THREAD

**Issue by FayezX** [DevExpress/devextreme-reactive#3168]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...

- [x] React Grid
- [ ] React Chart
- [ ] React Scheduler

### Description

Hi, is there any simple example that anyone can show me how to have the item transactions within a grouped expanded row set be paginated in any way instead of the infinite scrolling it seems to be doing by default?

**Comment by LazyLahtak** [comment]
Hi @FayezX,

Could you please clarify your goals? To help you with this issue, we need more information on your scenario.

**Comment by FayezX** [comment]
Hey @LazyLahtak , I am working with hundreds of thousands of rows of data and every table row data is controlled via responses coming in from services, which we want to be able to group by and basically looking at examples of Remote Grouping from -> https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/grouping/ 

This is all fine and I got it working after following documentation but I am noticing that after grouping by a column We get grouped data when we expand rows but did not find a way to paginate that grouped data we see so I can provide some limit for example 200 and next page click would trigger a service to get me next 200 for that expanded grouped data set. Since I am working with many rows, I have been looking for a way to paginate only the expanded subset of grouped rows grouped rows like how we can paginate the regular Table data seen here -> https://devexpress.github.io/devextreme-reactive/react/grid/docs/guides/paging/

In summary, I want to be able to group the table by column but be able to paginate the rows we see when we expand after a grouping.

Anything that can help will be appreciated, let me know if anything is unclear.

**Comment by LazyLahtak** [comment]
Thank you for the clarification.

The React Grid paging operates with Grid rows only.
As I understand it, the main problem is that the example from this [issue](https://github.com/DevExpress/devextreme-reactive/issues/3035) loads all rows for all groups, which is pretty expensive for you. Unfortunately, we do not have a more elegant solution for the Grouping feature in Lazy Loading mode.
I suggest that you use our [DataGrid for React](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/React/Light/) instead. It can load rows in groups piecemeal. You may wish to refer to this demo: [Data Grid. Remote Grouping](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/RemoteGrouping/React/Light/).

If you have any further questions, feel free to contact us. I will be happy to follow up.
